js/apps/system/_admin/aardvark/APP/frontend/js/templates/collectionsItemView.ejs

```diff
@@ -12,13 +12,8 @@
         <%= model.get('status') %>
       </div>
     <% } else if (model.get('status') === "loaded") { %>
-      <% if (!model.get('locked')) { %>
-        <div class="corneredBadge <%= model.get('status') %>">
-          <%= model.get('status') %>
-        </div>
-      <% } else { %>
+      <% if (model.get('locked')) { %>
         <div class="corneredBadge inProgress">
-          <%= model.get('status') %>
```

Comment:
Is this right? Won't this just show an empty badge then?

Reply:
It will display an unlabeled badge at least.
I don't know how it would look like, and I don't know if we can actually get into this state after reducing the number of collection statuses.
I can try to force this state manually just to see how it looks like, and then I will either remove the badge or leave it there if it still looks differently (the `inProgress` class may still modify the thing visually, but I don't know).

Reply:
After checking locally, it does look pretty ugly as the height expands weirdly when the div has no content, so I'd vote for simply not showing a badge at all or keeping the status in there if we still want to show the badge.

Reply:
Here's what the `loaded` one looks like without content for comparison:
![image](https://user-images.githubusercontent.com/1280922/127372576-85d2c2ce-dcf8-459f-a809-099799af29c7.png)

Reply:
I came to the same conclusion just now. PR is adjusted accordingly.